.cirrus.yml

```diff
@@ -0,0 +1,65 @@
+# This CI is used to test Arm cases. We can set the maximum 16 tasks.
+# The entire testing design is inspired from .github/workflows/compilers.yml.
+
+# By default, Cirrus mounts an empty volume to `/tmp`
+# which triggers all sorts of warnings like "system temporary path is world-writable: /tmp".
+# Lets workaround it by specifying a custom volume mount point.
+env:
+  CIRRUS_VOLUME: /cirrus-ci-volume
+
+task:
+  name: Arm64 Graviton2 / $CC
+  arm_container:
+    # We use the arm64 images at http://ghcr.io/ruby/ruby-ci-image .
+    image: ghcr.io/ruby/ruby-ci-image:$CC
+    # Define the used cpu core in each matrix task. We can use total 16 cpu
+    # cores in entire matrix. [cpu] = [total cpu: 16] / [number of tasks]
+    cpu: 8
+    # We can request maximum 4 GB per cpu.
+    # [memory per task] = [memory per cpu: 4 GB] * [cpu]
+    memory: 32G
+  env:
+    CIRRUS_CLONE_DEPTH: 50
+    optflags: '-O1'
+    debugflags: '-ggdb3'
+    RUBY_PREFIX: /tmp/ruby-prefix
+    RUBY_DEBUG: ci rgengc
+    RUBY_TESTOPTS: >-
+      -q
+      --color=always
+      --tty=no
+    matrix:
+      CC: clang-12
+      CC: gcc-11
+  id_script: id
+  set_env_script:
+    # Set `GNUMAKEFLAGS`, because the flags are GNU make specific. Note using
+    # the `make` environment variable used in compilers.yml causes some rubygems
+    # tests to fail.
+    # https://github.com/rubygems/rubygems/issues/4921
+    - echo "GNUMAKEFLAGS=-s -j$((1 + $CIRRUS_CPU))" >> $CIRRUS_ENV
+  print_env_script:
+    - echo "GNUMAKEFLAGS=$GNUMAKEFLAGS"
+  # Set any timezone to pass the tests.
+  set_timezone_script: sudo ln -snf /usr/share/zoneinfo/Japan /etc/localtime
+  dpkg-reconfigure_script: sudo dpkg-reconfigure --frontend noninteractive tzdata
```

Comment:
I noticed the following commands are not necessary. Perhaps the installed gems `timezone` and `tzinfo` do something to pass the tests. So I will remove the 2 commands.

```
-  # Set any timezone to pass the tests.
-  set_timezone_script: sudo ln -snf /usr/share/zoneinfo/Japan /etc/localtime
-  dpkg-reconfigure_script: sudo dpkg-reconfigure --frontend noninteractive tzdata
```